src/taxon.jl

```diff
@@ -80,3 +80,92 @@ function taxon(name::String;
    end
 
 end
+
+"""
+**Get information about a taxon at any level using taxonID**
+
+    taxon(id::Int)
+
+This function will look for a taxon by its taxonID in the GBIF
+reference taxonomy.
+
+Optional arguments are
+
+- `rank::Union{Symbol,Nothing}=:SPECIES` -- the rank of the taxon you want. This
+  is part of a controlled vocabulary, and can only be one of `:DOMAIN`,
+  `:CLASS`, `:CULTIVAR`, `:FAMILY`, `:FORM`, `:GENUS`, `:INFORMAL`, `:ORDER`,
+  `:PHYLUM,`, `:SECTION`, `:SUBCLASS`, `:VARIETY`, `:TRIBE`, `:KINGDOM`,
+  `:SUBFAMILY`, `:SUBFORM`, `:SUBGENUS`, `:SUBKINGDOM`, `:SUBORDER`,
+  `:SUBPHYLUM`, `:SUBSECTION`, `:SUBSPECIES`, `:SUBTRIBE`, `:SUBVARIETY`,
+  `:SUPERCLASS`, `:SUPERFAMILY`, `:SUPERORDER`, and `:SPECIES`
+
+- `strict::Bool=true` -- whether the match should be strict, or fuzzy
+
+- `verbose::Bool=false` -- whether the query should print out information about the reply (not usually useful)
+
+Finally, one can also specify other levels of the taxonomy, using  `kingdom`,
+`phylum`, `class`, `order`, `family`, and `genus`, all of which can either be
+`String` or `Nothing`.
+"""
+function mytaxon(id::Int;
+   rank::Union{Symbol,Nothing}=:SPECIES, strict::Bool=true, verbose::Bool=false,
```

Comment:
Again, everything from here to line 150 can be removed -- in the function with a taxon name, these are used to restrict the search; but as an ID will necessarily point to a single taxon, they are not needed.